hazelcast-client/src/test/java/com/hazelcast/client/discovery/AwsCloudDiscoveryTest.java

```diff
@@ -119,16 +123,25 @@ public void testAwsAddressProvider() {
                 .setTagKey(AWS_TEST_TAG)
                 .setTagValue(AWS_TEST_TAG_VALUE);
 
+
         ClientConfig clientConfig = new ClientConfig();
-        clientConfig.getNetworkConfig().setAwsConfig(clientAwsConfig);
+        String instanceName = null;
```

Comment:
This should be `String instanceName;`, otherwise a warning is created.

Reply:
done.